src/Agoda.Analyzers/RuleContent/AG0020AvoidReturningNullEnumerables.html

```diff
@@ -0,0 +1,48 @@
+﻿﻿<p>
+    Prevent returning null from a method / property whose return value implements IEnumerable.
+</p>
+
+<h2>Noncompliant Code Example</h2>
+<pre>
+public IEnumerable<int> GetPropertyIds(int hostId)
```

Comment:
Should this example be more generic since this is a public repo?

Reply:
Property -> Product fixed 👍